### ISSUE TITLE
ROADM "add_drop_osnr" affects result even in case where ROADM is intentionally not present in the network file.

### ISSUE THREAD

**Issue by ggrammel** [Telecominfraproject/oopt-gnpy#286]
step-1: Note there is no ROADM in the default network file ```edfa_example_network.json```

```
python transmission_main_example
<snip>
Transmission result for input power = 0.00dBm :
Transceiver Site_B
  OSNR ASE (0.1nm, dB):      32.03
  OSNR ASE (signal bw, dB):  27.95
  SNR total (signal bw, dB): 26.27
  SNR total (0.1nm, dB): 30.35
```

Step-2: editing eqpt_config.json changing ROADM parameter "add_drop_osnr" from 38 to 100.
```
python transmission_main_example.py 
<snip>
Transmission result for input power = 0.00dBm :
Transceiver Site_B
  OSNR ASE (0.1nm, dB):      33.30
  OSNR ASE (signal bw, dB):  29.21
  SNR total (signal bw, dB): 27.09
  SNR total (0.1nm, dB): 31.17
```

Summary:
If an equipment type is not present in the network description file, its parameters shall not alter the result of that network simulation.

Note: Thanks Jonas for uncovering this bug

**Comment by ojnas** [comment]
Already reported: #274

**Comment by ggrammel** [comment]
Closed as it is a duplicate of #274. The comment has been copied over.
